models/balancer/gnosis/balancer_gnosis_sources.yml

```diff
@@ -25,15 +25,15 @@ sources:
             name: evt_block_number
             description: 'Block number which processed the unique transaction hash'
           - name: poolAddress
-            description: 'Gnosis address for the liquidity pool used in transaction'
+            description: 'gnosis address for the liquidity pool used in transaction'
           - name: poolId
             description: 'Unique encoded identifier that refers to each pool'
           - name: specialization
             description: 'Pool specialization'
 
       - name: Vault_evt_Swap
         description: >
-          Decoded table related to the the contracts emitted by swaps in Balancer V2 pools.
+          Decoded table related to the contracts emitted by swaps in Balancer V2 pools.
```

Comment:
I assume this file was copied from other chains - could you fix the same typo on other sources yaml files?